src/sentry/taskworker/registry.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import annotations
```

Comment:
Should we call this file `namespace.py` and name the file which contains `TaskRegistry` with `registry.py` instead? Or are they going to live in the same file?

Reply:
They lived in the same file before. Lets see how big `TaskRegistry` gets and then we can split the files up if necessary.